src/technologydata/utils/carriers.txt

```diff
@@ -13,17 +13,21 @@
 # Format:
 # <carrier_name> = [<carrier_name>] [= <alias>]
 
+ammonia = [ammonia] = NH3
 carbon = [carbon] = C
 carbon_dioxide = [carbon_dioxide] = CO2
 carbon_monoxide = [carbon_monoxide] = CO
-coal = [coal]
+coal = [coal] = anthracite = hard_coal = black_coal
 diesel = [diesel]
+gasoline = [gasoline] = petrol
 jet_fuel_a1 = [jet_fuel_a1] = JETA1
 electricity = [electricity] = e = el
 hydrogen = [hydrogen] = H2
+lignite = [lignite] = lignite
 methane = [methane] = CH4
 methanol = [methanol] = CH3OH = MeOH
 natural_gas = [natural_gas] = NG
 nitrogen = [nitrogen] = N2
 oxygen = [oxygen] = O2
 water = [water] = H2O
+wood = [wood] = wood
```

Comment:
```suggestion
wood = [wood]
```

Everything after the `[<dimension>]` definition are alias definitions, no need to repeat "wood" as alias for "wood".

Get's rid of this warning in the tests:

<img width="752" height="43" alt="image" src="https://github.com/user-attachments/assets/ad76b4cc-98dd-4db9-a30f-fed56ad4237d" />